packages/radix/src/components/ToggleButton.tsx

```diff
@@ -52,35 +47,79 @@ export const ToggleButton = withTheme(
               borderBottomRightRadius: themeGet('radii.1')(props),
             },
             ':hover': {
-              borderColor: 'gray500',
               zIndex: 2, // TODO: Review in Primitives, this removes the overlapping
             },
             ':focus': {
               outline: 'none',
-              borderColor: 'blue500',
-              boxShadow: `0 0 0 1px ${themeGet('colors.blue500')(props)}`,
               zIndex: '5 !important', // TODO: Review in Primitives
               borderRadius: 1,
             },
           }),
-          { lineHeight: '1' },
+          variant({
+            size: {
+              0: {
+                height: 3,
+                padding: 0,
+                fontSize: 0,
+                minWidth: 4,
+                lineHeight: 1.1,
+                /**
+                 * TODO: Consider using box shadow for consistency
+                 *
+                 * This isn't too straightforward though. We can’t reliably center elements inside `ToggleButton` —
+                 * like same height icons — unless we remove the borders and use box-shadows instead, but also can’t
+                 * remove the borders and use box-shadows, because we use negative margins to compensate for adjacent
+                 * border.
+                 */
+                borderColor: 'transparent',
+                ':hover': {
```

Comment:
Can we keep the `&` in these pseudo classes? I think the rest of the codebase follows that syntax.

Reply:
> looks like it's missing in a few places in this file

Haha, that's where I got the idea from — _“must remove the ampersands, must be consistent with the rest of the file!”_ 

Done 😁